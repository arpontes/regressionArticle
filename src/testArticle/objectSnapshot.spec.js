test('criando um snapshot de um objeto', () => {
    const text = JSON.stringify({ id: 1, userName: "Andre", access: true });
    expect(text).toMatchSnapshot();
});