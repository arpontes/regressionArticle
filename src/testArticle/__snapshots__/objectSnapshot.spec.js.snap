// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`criando um snapshot de um objeto 1`] = `"{\\"id\\":1,\\"userName\\":\\"Andre\\",\\"access\\":true}"`;
